tests/test_client.py

```diff
@@ -2,8 +2,8 @@
 from typing import Union
 
 import pytest
-from pkg_resources import get_distribution
 from requests import Session
+import importlib_metadata
```

Comment:
Thanks for spotting that!

I would be ready to accept this PR, but the pre-commit hook `isort` is failing due to the non-standard ordering of imports. Please see [CONTRIBUTING.md](https://github.com/ArangoDB-Community/python-arango/blob/main/CONTRIBUTING.md) for more details.